stack/common.go

```diff
@@ -0,0 +1,31 @@
+package stack
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/url"
+)
+
+func ParseYAML(
+	file string,
```

Comment:
This doesn't read very well split over so many lines. How does it look on one line?

If there is a function being passed in - perhaps extract it to an interface?

Reply:
What do you mean by extract to an interface?

Reply:
Actually removing this entire file for now